index.md

```diff
@@ -65,10 +65,19 @@ In the meantime, publication of Internet-Draft documents can be tracked through
 * [Relative JSON Pointers](https://datatracker.ietf.org/doc/draft-bhutton-relative-json-pointer/)
 
 Internet-Drafts expire after six months, so our goal is to publish often enough to always have a set of unexpired drafts available.  There may be brief gaps as we wrap up each draft and finalize the text.
+</details>
 
-The intention, particularly for vocabularies such as validation which have been widely implemented, is to remain as compatible as possible from draft to draft.  However, these are still drafts, and given a clear enough need validated with the user community, major changes can occur.
+### Use of the _draft_ qualifier
 
-</details>
+Releases of the JSON schema specification and meta schemas are qualified as _draft_ primarily for historical reasons stemming from the relationship of this specification to IETF.
```

Comment:
Perhaps add that we are retaining the historical situation while we determine the best path forward, to avoid changing designation styles multiple times.  Or something along those lines.  This statement on its own makes it sound like it's basically inertia, and there is actually _slightly_ more to it than that.  If only slightly.

Reply:
If you want I can also change the phrasing of qualifier to designation, not sure what others think. I think "designation" may be slightly better than "qualifier".

Reply:
Yeah I had not consciously noticed that but I agree that designation works better.  Should we link the word designation to http://json-schema.org/specification-links.html#understanding-draft-names-and-numbers ?

Reply:
will do that

Reply:
Okay I added:

> The use of this designation is under review but will continue until this review process completes to avoid changing the designation style multiple times.

And changed "qualifier" to "designation".

Reply:
Looks good to me.